hypershift-operator/controllers/sharedingress/router_config.template

```diff
@@ -30,22 +30,31 @@ frontend dataplane-kas-svc
 
   {{- range .Backends }}
   acl is_{{ .Name }} var(sess.cluster_id) -m str {{ .ClusterID }}
+  acl is_{{ .Name }}_request_allowed src {{ .AllowedCIDRs }}
   {{- end }}
+
   {{- range .Backends }}
-  use_backend {{ .Name }} if is_{{ .Name }}
+  use_backend {{ .Name }} if is_{{ .Name }} is_{{ .Name }}_request_allowed
   {{- end }}
 
+  default_backend no-match
+
 # Frontends support any traffic coming from external DNS.
 frontend external-dns
   bind :::8443 v4v6
   tcp-request inspect-delay 5s
   tcp-request content accept if { req_ssl_hello_type 1 }
 
+  tcp-request content set-var(sess.src_ip) src
+
+  log-format "%{+Q}o\ src_ip = %[var(sess.src_ip)], %ci:%cp / %si:%sp -> %fi:%fp [%t] %ft %b/%s %Tw/%Tc/%Tt %B %ts %ac/%fc/%bc/%sc/%rc %sq/%bq"
+
   {{- range .ExternalDNSBackends }}
   acl is_{{ .Name }} req_ssl_sni -i {{ .HostName }}
+  acl is_{{ .Name }}_request_allowed src 91.47.40.124
```

Comment:
What is this IP? I think this is a typo from the original snippet

Reply:
I am not sure. I will remove it.